.github/workflows/unit_testing.yml

```diff
@@ -34,7 +34,8 @@ jobs:
 
     - name: Build and Test
       working-directory: ${{github.workspace}}
-      run: ./scripts/build.sh -p
+      shell: bash {0}
+      run: BOOST_ROOT="${{ steps.install-boost.outputs.BOOST_ROOT }}" ./scripts/build.sh -p
```

Comment:
Does the BOOST_ROOT just basically tell cmake where to find it?

Reply:
Yes, `BOOST_ROOT` specifies the source/root directory for the Boost libraries.